packages/ui/src/composables/useForm.ts

```diff
@@ -15,45 +17,18 @@ export const useFormPropsWithId = {
  * Create `readonly` and `disabled` BEM modifiers.
  * @param props component props
  * @param prefix string with which classes starts (and ends with form state BEM modifier)
+ * @returns computed Object/Array with classes which starts with `prefix` and ends with form state BEM modifier
  */
 export const useForm = <Prefix extends string>(
   props: ExtractPropTypes<typeof useFormProps>,
   prefix?: Prefix,
 ) => {
-  const computedClassesWithoutWarning = computed(() => ({
-    [`${prefix || ''}--disabled`]: props.disabled,
-    [`${prefix || ''}--readonly`]: props.readonly,
-  }) as Record<`${Prefix}--disabled` | `${Prefix}--readonly`, boolean>)
-
-  /**
-   * Computed Object with classes which starts with `prefix` and ends with form state BEM modifier
-   */
-  const computedClasses = computed(() => {
-    if (!prefix) {
-      console.warn('You must pass the @param "prefix" to the useForm hook!')
-    }
-
-    return computedClassesWithoutWarning.value
-  })
-
-  const computedClassesArrayWithoutWarning = computed(() => [
-    props.disabled && `${prefix || ''}--disabled`,
-    props.readonly && `${prefix || ''}--readonly`,
-  ].filter(Boolean) as Array<`${Prefix}--disabled` | `${Prefix}--readonly`>)
-
-  /**
-   * Computed Array with classes which starts with `prefix` and ends with form state BEM modifier
-   */
-  const computedClassesArray = computed(() => {
-    if (!prefix) {
-      console.warn('You must pass the @param "prefix" to the useForm hook!')
-    }
-
-    return computedClassesArrayWithoutWarning.value
-  })
-
-  return {
-    computedClasses,
-    computedClassesArray,
-  }
+  const {
+    computedBemClassesObject: computedClasses,
+    computedBemClassesArray: computedClassesArray,
+  } = useBem(
+    prefix, computed(() => pick(props, ['disabled', 'readonly'])),
+  )
+
+  return { computedClasses, computedClassesArray }
```

Comment:
I'm wondering how many times we will rename these names. How about having two different hooks? useBemClassObject useBemArray?
```suggestion
  return computedClasses
```

or

```
const wrapperClass = useBem()
...
wrapperClass // Returns object

// or wrapperClass.asString
// or wrapperClass.asArray
```

The last example will make you a giga 🧠 if you implement it.